lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -178,19 +174,47 @@ final class AddComicViewController: UIViewController {
         finishButton.tintColor = LucienTheme.dark
     }
 
+    private func darkenTextFieldBottomBorder(textField: UITextField) {
+        guard let bottomBorderSubLayer = textField.layer.sublayers else { return }
+        let bottomBorder = bottomBorderSubLayer[0] as CALayer
+        bottomBorder.backgroundColor = LucienTheme.dark.cgColor
+    }
+
+    @objc private func seriesTitleEditingChanged(_ textField: UITextField) {
+        seriesTitleWarningLabel.isHidden = true
+        darkenTextFieldBottomBorder(textField: textField)
+        checkIfAllRequiredFieldsAreFilled()
+    }
+
+    @objc private func storyTitleEditingChanged(_ textField: UITextField) {
+        storyTitleWarningLabel.isHidden = true
+        darkenTextFieldBottomBorder(textField: textField)
+        checkIfAllRequiredFieldsAreFilled()
+    }
+
     @objc private func releaseDateEditingChanged(_ textField: UITextField) {
         guard let text = textField.text else { return }
         if text.characters.count > 4 {
             textField.deleteBackward()
         }
     }
 
-    @objc func doneButtonTapped() {
+    @objc private func doneButtonTapped() {
         view.endEditing(true)
         deregisterFromKeyboardNotifications()
     }
 
-    @IBAction func addCoverButtonTapped(_ sender: UIButton) {
+    private func showWarningLabel(label: UILabel, textField: UITextField, sublayers: [CALayer]) {
```

Comment:
Why are the sublayers being passed in here? Don't they exist on the textfield. If you implement a subclass for these text fields this comment will be moot.